easybuild/easyconfigs/r/Rosetta/Rosetta-3.12-foss-2018b.eb

```diff
@@ -0,0 +1,17 @@
+name = 'Rosetta'
+version = '3.12'
+
+homepage = 'https://www.rosettacommons.org'
+description = """Rosetta is the premier software suite for modeling macromolecular structures. As a flexible,
+multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and
+nucleic acids."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+sources = ['%(namelower)s_src_%(version)s_bundle.tgz']
```

Comment:
@lexming Don't we need a comment here about the licensing requirement prior to download?

Reply:
good idea, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/11256/commits/4375c5d4d3faa5abc77992068297fa995c00169e